tests/foreman/api/test_reporttemplates.py

```diff
@@ -707,3 +711,98 @@ def test_positive_generate_job_report(setup_content, target_sat, rhel7_contentho
     )
     assert res[0]['Host'] == rhel7_contenthost.hostname
     assert '/root' in res[0]['stdout']
+
+
+@pytest.mark.tier2
+@pytest.mark.rhel_ver_match(r'^(?!6$)\d+$')
```

Comment:
`@pytest.mark.rhel_ver_list([7, 8, 9])` wouldn't this be clearer?

Reply:
It would be, but this way we have to go through all tests and add RHEL 10 once we start testing it.
I was already thinking about some helper function that would return a list of all supported RHEL versions that we would use as a value for `rhel_ver_list`. 
Is that a good idea?